nf_core/modules/lint/main_nf.py

```diff
@@ -211,23 +211,18 @@ def check_process_section(self, lines):
         self.warned.append(("process_standard_label", "Process label unspecified", self.main_nf))
 
     for l in lines:
-        l = l.strip()
-        l = l.replace('"', "")
-        l = l.replace("'", "")
         if re.search("bioconda::", l):
             bioconda_packages = [b for b in l.split() if "bioconda::" in b]
+        l = l.strip()
+        l = l.strip(" '\"")
```

Comment:
No need for the first one - the second call includes a space as one of the characters that we're removing..
```suggestion
        l = l.strip(" '\"")
```